README.md

```diff
@@ -0,0 +1,209 @@
+# Proposing and discussing changes
+
+## Introduction
+
+The repository first objective is to give a place where community memebers can document their design ideas for WoT projects. 
+The structure of this repository is heavily inspired by the [golang proposal system](https://github.com/golang/proposal).
+
+Each design change must be discussed on a proposal issue. When needed, a document expressing the change motivations is merged inside this repository and refrenced to the issue with a precise [naming schema](#design-documents).
+
+Users must adhere to the [code of conduct](#code-of-conduct) of the community when discussing a proposal. Partecipating to the project evolutions needs to be a pleasant experience for everyone. We pursue a friendly, welcoming environment where everyone can express their ideas, always feeling at home. 
+
+This document will describe the whole proposal workflow, from the opening of an issue, to the acceptance or refusal of design changes through discussion and engagement of the community. 
+
+## What should be considered as a 'Proposal' ?
+
+Proposals are essentially design changes of any of the wot-open-source-project repositories.
```

Comment:
design changes or proposals for a new tool...

Reply:
Yeah it is more readable and concise. Thank you